### PR TITLE
[FIX] Network Explorer: settings

### PR DESCRIPTION
**Pull request by jerneju**
##### Issue
Settings does not work. Some dead code and some things not imported.

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: orangecontrib/network/widgets/OWNxExplorer.py [hunk 1]
```diff
@@ -435,33 +432,22 @@ def _set_combos(self):
...
-            model = self.attListBox.model()
-            for i in range(self.attListBox.count()):
-                if str(self.attListBox.item(i).text()) in lastLabelColumns:
-                    selection.append(QItemSelectionRange(model.index(i, 0)))
-            selmodel = self.attListBox.selectionModel()
-            selmodel.select(selection, selmodel.Select | selmodel.Clear)
-        else:
-            self.attListBox.selectionModel().clearSelection()
+        for columns, box in ((self.attrs_label, self.attListBox),
+                               (self.attrs_tooltip, self.tooltipListBox)):
```
Comment (kernc) [inline orangecontrib/network/widgets/OWNxExplorer.py]: Looks misaligned.

File: orangecontrib/network/widgets/OWNxExplorer.py [hunk 2]
```diff
@@ -66,9 +68,10 @@ class Outputs:
         highlighted = Output("Highlighted items", Table)
         remaining = Output("Remaining items", Table)
 
-    settingsList = ["lastVertexSizeColumn", "lastColorColumn",
-                    "lastLabelColumns", "lastTooltipColumns",]
-    # TODO: set settings
+    attr_size = Setting(None, schema_only=True)
+    attr_color = Setting(None, schema_only=True)
+    attrs_label = Setting({})
+    attrs_tooltip = Setting({})
```
Comment (kernc) [inline orangecontrib/network/widgets/OWNxExplorer.py]: Move below to where other Settings are?

Comment (kernc) [reply orangecontrib/network/widgets/OWNxExplorer.py]: "attr" settings should be `ContextSetting`?

File: orangecontrib/network/widgets/OWNxExplorer.py [hunk 3]
```diff
@@ -408,67 +408,34 @@ def send_data(self):
...
-            self.tooltipListBox.selectionModel().clearSelection()
+        self.color_model[:] = [None] + [v for v in self.graph_attrs if v.is_primitive]
+        self.size_model[:] = [None] + [v for v in self.graph_attrs if v.is_continuous]
+        self.size_combo.setDisabled(not self.graph_attrs)
+        self.color_combo.setDisabled(not self.graph_attrs)
+        self.set_node_sizes()
+        self.set_node_colors()
+
+        for columns, box in ((self.attrs_label, self.attListBox),
+                               (self.attrs_tooltip, self.tooltipListBox)):
```
Comment (kernc) [inline orangecontrib/network/widgets/OWNxExplorer.py]: Looks misaligned.

Comment (jerneju) [reply orangecontrib/network/widgets/OWNxExplorer.py]: I think we should enable lint.

**Pull request merged**